apiserver/facades/client/client/client.go

```diff
@@ -263,14 +270,43 @@ func (c *Client) WatchAll() (params.AllWatcherId, error) {
 	if err != nil {
 		return params.AllWatcherId{}, errors.Trace(err)
 	}
-	watchParams := state.WatchParams{IncludeOffers: isAdmin}
-
-	w := c.api.stateAccessor.Watch(watchParams)
+	_ = isAdmin
```

Comment:
What is this for?

Reply:
and should it be about being a superuser vs an admin?

Reply:
It was a carry-over from before I implemented the filtering, but I didn't want to forget to add it.

It can be removed now. I'll add that in to my cleanup branch.